corehq/apps/app_manager/app_translations/app_translations.py

```diff
@@ -645,15 +645,16 @@ def has_translation(row_, langs):
             markdowns[label_id] = markdowns[label_id] or had_markdown(text_node)
     # skip labels that have no translation provided
     skip_label = set()
-    for row in rows:
-        if not has_translation(row, app.langs):
-            skip_label.add(row['label'])
-    for label in skip_label:
-        msgs.append((
-            messages.error,
-            "You must provide at least one translation" +
-            " for the label '%s' in sheet '%s'" % (label, sheet.worksheet.title)
-        ))
+    if form.is_registration_form():
```

Comment:
how come this is only for registration forms?

Reply:
@pr33thi 
the failure only happens for registration forms
https://github.com/dimagi/commcare-hq/blob/52897b9680906ff5b4ad0954faac779bcdc249ab/corehq/apps/app_manager/util.py#L163 when `form.get_questions([app.default_language])` is called.

Reply:
cool, thanks